portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1281,57 +1274,81 @@ definition {
 			virtualHost = "www.baker.com",
 			webId = "www.baker.com");
 
+		MyAccount.openMyAccountAdmin(baseURL = "http://www.test.com:8080");
```

Comment:
@weller0246 Could you please change this URL to http://localhost:8080, according to the test case terminolodgy.